watchtower/wtclient/session_negotiator.go

```diff
@@ -126,7 +126,9 @@ func newSessionNegotiator(cfg *NegotiatorConfig) *sessionNegotiator {
 	features := []lnwire.FeatureBit{
 		wtwire.AltruistSessionsRequired,
 	}
-	if cfg.Policy.IsAnchorChannel() {
+	if cfg.Policy.IsTaprootChannel() {
+		features = append(features, wtwire.TaprootCommitRequired)
```

Comment:
Similar to other refactors in this PR, could have a method on `Policy` that returns the proper feature bit.

Reply:
Or something similar to `TypeFromChannel` but returns the wtwire bits.